virttest/tests/unattended_install.py

```diff
@@ -1217,7 +1217,7 @@ def attempt_to_log_useful_files(test, vm):
                             fd_dst.write("Unknown exception while getting "
                                          "content: %s" % details)
                             failures = True
-            for cmd in ["journalctl --no-pager"]:
+            for cmd in ["journalctl --no-pager", "udevadm info --export-db"]:
```

Comment:
@ldoktor   cmd  run once is enough, That code should not be in the outermost loop

Reply:
In case there are no failures, the commands will only be executed just once. Otherwise it makes sense to execute them multiple times as it's likely they failed to execute as well (this usually happens when net becomes unusable).

Reply:
This may be a question I've asked before (during the initial feature review), but doesn't this make it possible to overwrite `dst` file in a different iteration of the loop (over sessions)? Shouldn't this:
1. Check if `dst` exists
2. If not, run the command
3. If successful, create the `dst` file with the content from the command

Reply:
No, each iteration uses a different direcotry (see line 1221). In case of a failure it might be useful to have multiple copies and usually it PASSes so one ends up with a sigle dir.